tools/dependencies/known-oap-backend-dependencies.txt

```diff
@@ -61,17 +61,26 @@ httpclient-4.5.13.jar
 httpcore-4.4.13.jar
 httpcore-nio-4.4.13.jar
 influxdb-java-2.15.jar
+iotdb-session-0.13.0-SNAPSHOT.jar
+iotdb-thrift-0.13.0-SNAPSHOT.jar
+istack-commons-runtime-4.0.0.jar
 jackson-annotations-2.12.2.jar
 jackson-core-2.12.2.jar
 jackson-databind-2.12.2.jar
 jackson-datatype-guava-2.9.10.jar
 jackson-datatype-jdk8-2.9.10.jar
 jackson-module-afterburner-2.12.2.jar
 jackson-module-kotlin-2.8.8.jar
+jakarta.activation-2.0.0.jar
+jakarta.xml.bind-api-3.0.0.jar
```

Comment:
FYI @kezhenxu94 I think our maven cache may cause some issues when we use `SNAPSHOT` for draft PR.

![image](https://user-images.githubusercontent.com/5441976/140316608-3007b1a0-91e5-4cc0-a0ee-867ccfef8eff.png)

I and @LIU-WEI-git both packaged locally, these jars exist, but CI checks on dependencies say otherwise.
```
@@ -74,2 +72,0 @@
-jakarta.activation-2.0.0.jar
-jakarta.xml.bind-api-3.0.0.jar
```

I don't think we have to fix this, as we never would accept a PR using snapshot dependency. Just find out this curious case.

Reply:
I think it is your local cache that cause this issue, because when maven downloads a SNAPSHOT package it will firstly check whether it exists in your local repository (typically `~/.m2/repository`), and if exists it will be used directly, but SNAPSHOT can be overrode in remote repository so you might be using an old revision, as I check the latest revision of this iotdb-session SNAPSHOT is `iotdb-session-0.13.0-20211102.115555-1.jar Tue Nov 02 14:07:09 UTC 2021  67297` so you might want to update the local SNAPSHOT and repackage to see the changes, generally adding `-U` will do the trick (`./mvnw -U ....`)

Reply:
`snapshot` is not always using the local repository, it just has an expired time. 

I did local check again with your recommend `-U`, still get the same result.
![image](https://user-images.githubusercontent.com/5441976/140334095-fae6eb1f-3d18-4aa6-bf3c-d1e548a08f53.png)

I checked my iotdb dependencies, they are the latest.
![image](https://user-images.githubusercontent.com/5441976/140334535-cc7bf43c-b0bc-4913-aa9e-c99a71513216.png)

My question is not about IoTDB session version, I was asking about `jakata*`, which CI says they don't exist.

Reply:
I build the package and didn't find these 2 jar as well, the same as what we see in GitHub Actions.

```diff
@@ -74,2 +72,0 @@
-jakarta.activation-2.0.0.jar
-jakarta.xml.bind-api-3.0.0.jar
```

Reply:
I also don't find any related jars in the dependency tree of new module storage-iotdb-plugin

Reply:
OK, it seems need to delete some local jars. Anyway, this would not be an issue after official release.